ethereum/contracts/MMRVerification.sol

```diff
@@ -92,17 +93,7 @@ contract MMRVerification {
         // TODO: Make this more elegant
         bytes32 rightPeaksHash;
         if (numRightPeaks > 0) {
-            if (numRightPeaks == 1) {
```

Comment:
so the only real logic change is removing these 2 cases and just doing a simple one-line bag for both cases?

Reply:
Exactly